README.md

```
@@ -219,7 +219,7 @@ See [LICENSE] for details.
 [contributor license agreement]: https://github.com/toeverything/affine/edit/canary/.github/CLA.md
 [stars-icon]: https://img.shields.io/github/stars/toeverything/AFFiNE.svg?style=flat&logo=github&colorB=red&label=stars
 [codecov]: https://codecov.io/gh/toeverything/affine/branch/canary/graphs/badge.svg?branch=canary
-[node-version-icon]: https://img.shields.io/badge/node-%3E=18.16.1-success
+[node-version-icon]: https://img.shields.io/badge/node-%3E=22.12.0%20%3C23.0.0-success
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "node-version-icon"

(MD053, link-image-reference-definitions)